src/components/gui/gui.jsx

```diff
@@ -26,7 +26,7 @@ const addExtensionMessage = (
     <FormattedMessage
         defaultMessage="Extensions"
         description="Button to add an extension in the target pane"
-        id="targetPane.addExtension"
+        id="gui.targetPane.addExtension"
```

Comment:
The convention says `repo.component.name`. Does that make sense when the component is gui? Should this be `gui.gui.addExtension`?